sdk/identity/identity/src/credentials/managedIdentityCredential/imdsMsi.ts

```diff
@@ -43,14 +45,15 @@ function prepareRequestOptions(resource?: string, clientId?: string): RequestPre
     queryParameters.client_id = clientId;
   }
 
+  const query = qs.stringify(queryParameters);
+
   return {
-    url: process.env.AZURE_POD_IDENTITY_TOKEN_URL ?? imdsEndpoint,
```

Comment:
Makes me nervous that we suddenly lost this env var.

Reply:
Thank you for catching this!